test/oidc-strategy.ts

```diff
@@ -33,7 +33,7 @@ describe("Integration", () => {
       const response = await server.inject({
         method: "GET",
         url: "/auth-check",
-        headers: { Authorization: `Bearer ${token} ` },
+        headers: { Authorization: `Bearer ${token.toString()} ` },
```

Comment:
token comes back as a `string` but it's typed as an `object` in node-jose. I do not understand why. This is just a test so i'm just calling `.toString()` on it instead of going through the trouble of type guarding this.